src/app/services/get-data.service.ts

```diff
@@ -42,7 +46,28 @@ export class GetDataService {
                 });
             });
     }
+
+    renderCardContent(postId, curPost): void {
+        this.db
+            .collection('Posts')
+            .get()
+            .pipe(takeUntil(this.destroy$))
+            .subscribe(snapshot => {
+                snapshot.docs.forEach(doc => {
+                    if (postId === doc.data().postId) {
+                        const post = doc.data();
+                        Object.assign(curPost, post);
```

Comment:
wat????

Reply:
What is the reason to assign? 
any function should not do any side effect onto provided params